DTS-STN/secure-client-hub
# Update grapqhl queries and mappers for SP18

## [ADO-146311](https://dev.azure.com/VP-BD/DECD/_workitems/edit/146311)

Fixed [AB#146311](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/146311)

### Changelog
fix:updated GraphQL queries and mappers for AEM SP18

### Description of proposed changes:
This PR updates our GraphQL queries and mappers so they work with the new update to AEM. Upon upgrading to SP18, AEM does away with the option to manually name our API calls (ie `schPagev1ByPath`) and does it automatically, which has a specific casing convention. This PR simply updates our queries and mappers to follow that new convention

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Ensure app works as expected

### Additional Notes
Persisted queries are now available so we should be making use of them in another PR. This PR only serves to get everything back up and running.
